<ui version="4.0" >
 <class>QUTaskDialog</class>
 <widget class="QDialog" name="QUTaskDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Edit Rename Task</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../../resources/uman.qrc" >
    <normaloff>:/control/tasks_add.png</normaloff>:/control/tasks_add.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="autoFillBackground" >
      <bool>true</bool>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="infoIconLbl" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="pixmap" >
         <pixmap resource="../../resources/uman.qrc" >:/marks/information.png</pixmap>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="infoLbl" >
        <property name="text" >
         <string>...</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" >
     <property name="spacing" >
      <number>5</number>
     </property>
     <property name="margin" >
      <number>5</number>
     </property>
     <item>
      <layout class="QGridLayout" name="gridLayout_2" >
       <property name="spacing" >
        <number>5</number>
       </property>
       <item row="0" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>&amp;Description</string>
         </property>
         <property name="buddy" >
          <cstring>descriptionEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="descriptionEdit" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>T&amp;oolTip</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="buddy" >
          <cstring>toolTipEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QTextEdit" name="toolTipEdit" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <property name="tabChangesFocus" >
          <bool>true</bool>
         </property>
         <property name="tabStopWidth" >
          <number>80</number>
         </property>
         <property name="acceptRichText" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>&amp;Icon</string>
         </property>
         <property name="buddy" >
          <cstring>iconCombo</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="iconCombo" >
           <property name="frame" >
            <bool>true</bool>
           </property>
           <item>
            <property name="text" >
             <string>Cup</string>
            </property>
            <property name="icon" >
             <iconset>
              <normaloff/>
             </iconset>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>&amp;Group</string>
         </property>
         <property name="buddy" >
          <cstring>groupSpin</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>5</number>
         </property>
         <item>
          <widget class="QSpinBox" name="groupSpin" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="exclusiveChk" >
           <property name="text" >
            <string>Exclusive task</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout" >
       <property name="spacing" >
        <number>5</number>
       </property>
       <item row="0" column="0" >
        <widget class="QLabel" name="label_7" >
         <property name="font" >
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text" >
          <string>&amp;Target</string>
         </property>
         <property name="margin" >
          <number>5</number>
         </property>
         <property name="buddy" >
          <cstring>targetCombo</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="targetCombo" >
           <property name="toolTip" >
            <string>The target specifies the object on which the renaming will be done.</string>
           </property>
           <item>
            <property name="text" >
             <string>mp3</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>68</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_6" >
         <property name="font" >
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text" >
          <string>&amp;Schema</string>
         </property>
         <property name="margin" >
          <number>5</number>
         </property>
         <property name="buddy" >
          <cstring>schemaEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <property name="spacing" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="schemaEdit" >
           <property name="minimumSize" >
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="font" >
            <font>
             <family>Lucida Console</family>
             <pointsize>9</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="addDataBtn" >
           <property name="text" >
            <string>...</string>
           </property>
           <property name="icon" >
            <iconset resource="../../resources/uman.qrc" >
             <normaloff>:/marks/add.png</normaloff>:/marks/add.png</iconset>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="removeDataBtn" >
           <property name="text" >
            <string>...</string>
           </property>
           <property name="icon" >
            <iconset resource="../../resources/uman.qrc" >
             <normaloff>:/marks/delete.png</normaloff>:/marks/delete.png</iconset>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>16</width>
             <height>22</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <item>
        <widget class="QUTaskDataTable" name="dataTable" >
         <property name="editTriggers" >
          <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
         </property>
         <property name="alternatingRowColors" >
          <bool>true</bool>
         </property>
         <property name="selectionMode" >
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior" >
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="gridStyle" >
          <enum>Qt::DashLine</enum>
         </property>
         <property name="cornerButtonEnabled" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="moveUpBtn" >
           <property name="minimumSize" >
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="text" >
            <string>...</string>
           </property>
           <property name="icon" >
            <iconset resource="../../resources/uman.qrc" >
             <normaloff>:/bullets/arrow_up_small.png</normaloff>:/bullets/arrow_up_small.png</iconset>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="moveDownBtn" >
           <property name="minimumSize" >
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="text" >
            <string>...</string>
           </property>
           <property name="icon" >
            <iconset resource="../../resources/uman.qrc" >
             <normaloff>:/bullets/arrow_down_small.png</normaloff>:/bullets/arrow_down_small.png</iconset>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>13</width>
             <height>44</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_3" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>5</number>
     </property>
     <property name="margin" >
      <number>5</number>
     </property>
     <item>
      <widget class="QPushButton" name="saveAsBtn" >
       <property name="text" >
        <string>Save As...</string>
       </property>
       <property name="icon" >
        <iconset resource="../../resources/uman.qrc" >
         <normaloff>:/control/save_as.png</normaloff>:/control/save_as.png</iconset>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveBtn" >
       <property name="text" >
        <string>Save</string>
       </property>
       <property name="icon" >
        <iconset resource="../../resources/uman.qrc" >
         <normaloff>:/control/save.png</normaloff>:/control/save.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn" >
       <property name="text" >
        <string>Cancel</string>
       </property>
       <property name="icon" >
        <iconset resource="../../resources/uman.qrc" >
         <normaloff>:/marks/cancel.png</normaloff>:/marks/cancel.png</iconset>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QUTaskDataTable</class>
   <extends>QTableWidget</extends>
   <header>QUTaskDataTable.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/uman.qrc" />
 </resources>
 <connections/>
</ui>
